### PR TITLE
Fixes #31966 - Add warning for foreman-rake permissions:reset task

### PR DESCRIPTION
**Pull request by Dyrkon**
Added warning to foreman-rake permissions:reset task.

### CODE REVIEW

File: lib/tasks/reset_permissions.rake [hunk 1]
```diff
@@ -22,6 +22,7 @@ namespace :permissions do
       user.password = password
       if user.save
         puts "Reset to user: #{user.login}, password: #{password}"
+        puts "WARNING: You need to change the password in /root/.hammer/cli.modules.d/foreman.yml manually!"
```
Comment (ofedoren) [inline lib/tasks/reset_permissions.rake]: Well, this is true for the `root` user. But since this task allows reseting of password for any user, this message should be more general.

Comment (Dyrkon) [reply lib/tasks/reset_permissions.rake]: Yes, thank you for the comment.

File: lib/tasks/reset_permissions.rake [hunk 2]
```diff
@@ -22,6 +22,7 @@ namespace :permissions do
       user.password = password
       if user.save
         puts "Reset to user: #{user.login}, password: #{password}"
+        puts "WARNING: You need to change the password in foreman.yml manually!"
```
Comment (ofedoren) [inline lib/tasks/reset_permissions.rake]: Looks better, but I'd rephrase it a bit. What about:
```suggestion
        puts "WARNING: You need to change the password for hammer in ~/.hammer/cli.modules.d/foreman.yml manually!"
```
?

Comment (stejskalleos) [reply lib/tasks/reset_permissions.rake]: What if Foreman user doesn't have same name as in Linux account?

Comment (ofedoren) [reply lib/tasks/reset_permissions.rake]: True, updated.

Comment (Dyrkon) [reply lib/tasks/reset_permissions.rake]: So how does this version reflect the root location? From what @stejskalleos told me, there is more than one location where the yaml file can be present. Should there be all of them?

Comment (ofedoren) [reply lib/tasks/reset_permissions.rake]: Well, if the user is root then `~/` gets translated into the root directory, doesn't it?

There are few places, yes. The main one is in `/etc/hammer`, but there should live settings which can be re-used by different users. Per user settings (such as their passwords) should live in their home `~/.hammer` directories.

Although I'm not sure if we force those directories, it's recommended.

Comment (Dyrkon) [reply lib/tasks/reset_permissions.rake]: I forgot about that. Thank you. I will implement it according to your suggestion.

**Review by ofedoren — APPROVED** [review]
Thanks, @Dyrkon, nothing more from my side :)

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, nice & clean.

Thanks @Dyrkon & @ofedoren

**Comment by upadhyeammit** [comment]
cped for 3.4-stable c64bc2f0805ebea396c34fd1d1011e75d8499737
